webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -166,7 +173,21 @@ export default class TicketPopover extends React.PureComponent<Props, State> {
             return null;
         }
 
-        const {ticketDetails} = this.state;
+        const {ticketDetails, error} = this.state;
+        if (error) {
+            return (
+                <div className='jira-issue-tooltip jira-issue-tooltip-error'>
+                    <span
+                        className='jira-issue-error-icon fa fa-exclamation-triangle'
```

Comment:
Use the icon showed the designs

Reply:
The icon used in the designs is part of the pro version of font awesome. I couldn't find a way to use it in our code. Please let me know any work around for this.

Reply:
You can use an svg for this